lib/vrt/map.rb

```diff
@@ -43,9 +40,16 @@ def categories
 
     private
 
+    def valid_identifier?(vrt_id)
+      # At least one string of lowercase or _, plus up to 2 more with stops
+      @_valid_identifiers[vrt_id] ||= vrt_id =~ /other|\A[a-z_]+(\.[a-z_]+){0,2}\z/
```

Comment:
Just figured I'd configure the `other` into the regex, lmk if you don't like that

Reply:
Nah, I dig it

Reply:
I personally find regex super confusing and avert my eyes at all costs.   not totally against it though.

Reply:
its is nice to have it all handled by one case.

Reply:
I agree in general, but at least its just a single literal of `other` and then regex horror show